geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+structure of CS2 dimer. J Chem Phys 134(14):144306/1-144306/5
+doi:10.1063/1.3578177
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C = C     3.539(7)
+C = C = S 90
+
+NAME = C3Ge2
+# Point Group: Dooh 
+# Shape: Linear
+# Bond lengths: re (angstroms)
+# Source paper: Thorwirth S, Lutter V, Javed AJ, Gauss J, Giesen TF (2016) Gas-phase spectroscopic detection and structural
+elucidation of carbon-rich group 14 binary clusters: Linear GeC3Ge. J Phys Chem A 120(2):254-259
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C ≡ Ge 1.7695(1)
+C = C  1.2893
+
+# C = C 1.2893 assumed at the CCSD(T)/cc-pwCVQZ value
+
+NAME = C3Pd
+# Point Group: Coov 
+# Shape: Linear
+# Comment: The molecule was found to be linear or quasilinear.
+# Bond lengths: r0 (angstroms)
+# Source paper: Bittner DM, Zaleski DP, Tew DP, Walker NR, Legon AC (2016) Highly unsaturated platinum and palladium
+carbenes PtC3 and PdC3 isolated and characterized in the gas phase. Angew Chem 128(11):3832-3835; Angew
+Chem Int Ed Engl 55(11):3768-3771
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Pd   = C(1) 1.79898(4)
+C(1) = C(2) 1.3009
+C(2) = C(3) 1.2789
+
+# C(1) = C(2) 1.3009 and C(2) = C(3) 1.2789 fixed to the value calculated at the level CCSD(T)_ae/aug-cc-pwCV5Z-PP(for Pd) and corrected for r0 ‒ re
+difference assumed to that for PtC3. 
+
+NAME = C3Pt
+# Point Group: Coov 
+# Shape: Linear
+# Comment: The molecule was found to be linear or quasilinear.
+# Bond lengths: r0 (angstroms)
+# Source paper: Bittner DM, Zaleski DP, Tew DP, Walker NR, Legon AC (2016) Highly unsaturated platinum and palladium
+carbenes PtC3 and PdC3 isolated and characterized in the gas phase. Angew Chem 128(11):3832-3835; Angew
+Chem Int Ed Engl 55(11):3768-3771
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Pt   = C(1) 1.7315(14)
+C(1) = C(2) 1.2993(19)
+C(2) = C(3) 1.2759(11)
+
+NAME = C4HNO 2012
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Kang L, Novick SE (2012) The microwave spectra of the weakly bound complex between carbon monoxide and
+cyanoacetylene, OC H–C≡C–C≡N. J Mol Spectrosc 276-277:10-13
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C(4)...H        2.65315(27)
+C(4)...D        2.64962(27)
+OC4    HC1C2C3N 6.2031
+
+NAME = C4HNO2 2017
+# Point Group: Coov 
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: Kang L, Davis P, Dorell I, Li K, Oncer O, Wang L, Novick SE, Kukolich SG (2017) Rotational spectrum and
+structure of the T-shaped cyanoacetylene carbon dioxide complex, HCCCN⋅⋅⋅CO2. J Mol Spectrosc 342(6):62-72
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+OCO NCCCH 4.8311(3)
+ϕ         10(1)
+φ         5.4(5)
+
+# ϕ: Angle of torsional oscillation about the a axis, defined as the angle between this axis and the HCCCN axis
+# φ: Angle of torsional oscillation about an axis parallel to b one (bꞌ), defined as the angle between bꞌ and the
+O=C=O axis
```

Comment:
Hopefully these angle descriptions can be built into our notation/format instead.